backend/apps/medical_catalog/api/views/labtest_views.py

```
@@ -24,22 +25,22 @@ class LabTestViewSet(viewsets.ModelViewSet):
 
     def create(self, request, *args, **kwargs):
```

Ruff (0.14.3)

Unused method argument: `args`

(ARG002)